packages/hoppscotch-cli/src/utils/mutators.ts

```diff
@@ -52,7 +52,21 @@ const getValidRequests = (
 export const toFormData = (values: FormDataEntry[]) => {
   const formData = new FormData();
 
-  values.forEach(({ key, value }) => formData.append(key, value));
+  values.forEach(({ key, value, contentType }) => {
+    if (contentType) {
+      formData.append(
+        key,
+        new Blob([value], {
+          type: contentType,
+        }),
+        key
+      );
+
+      return;
+    }
+
+    formData.append(key, value);
+  });
```

Comment:
Currently, specifying a content type of `text/plain` for an entry is still considered a file being wrapped in `Blob`. Try sending a request to https://httpbin.org/post with an entry in the above format and observe the entry listed under the key, `files` and not `form`. Could you investigate this?

Reply:
Discussed separately, this is the intended behaviour while specifying custom content types.